app/views/devise/registrations/new.html.erb

```diff
@@ -16,8 +17,8 @@
       
       <div class="field">
         <label><b><%= f.label :gender %></b></label><br />
-        <div><%= f.check_box :is_female, {}, true %> <%= f.label :is_female, "Female" %> </div>
-        <div><%= f.check_box :is_female, {}, false %> <%= f.label :is_female, "Male" %> </div>
+        <div><%= f.check_box :gender, {}, true %> <%= f.label :gender, "Female" %> </div>
```

Comment:
we have is_female above and gender here.

Reply:
I have fixed this.